target/linux/ramips/image/mt7620.mk

```diff
@@ -362,6 +362,20 @@ define Device/elecom_wrh-300cr
 endef
 TARGET_DEVICES += elecom_wrh-300cr
 
+define Device/fon_fon2601
+  MTK_SOC := mt7620a
+  IMAGE_SIZE := 15936k
+  DEVICE_VENDOR := Fon
+  DEVICE_MODEL := FON2601
+  DEVICE_PACKAGES := kmod-mt76x2 kmod-usb2 kmod-usb-ohci
+  KERNEL_INITRAMFS := $$(KERNEL) | fonfxcimage
+  IMAGE/sysupgrade.bin := append-kernel | append-rootfs |\
+			fonfxcimage |\
+			pad-rootfs | append-metadata | check-size $$$$(IMAGE_SIZE)
+  SUPPORTED_DEVICES += fon2601
```

Comment:
If you add a new device, you do not need to add SUPPORTED_DEVICES, as the default will be the DTS compatible string.
Just remove this line.

Reply:
Removed. done.